### ISSUE TITLE
iCal File download error

### ISSUE THREAD

**Issue by lorddoumer** [derhansen/sf_event_mgt#236]
Hi,

I really like your extension but can't get the ics download to work, everything else is working fine.

The ics file is generated in the _temp_ directory but the downloadlink in the frontend doesn't seem to link to the destination. Firefox opens the download-windows but says:
"path/to/xxx.ics.part could not be safed because the source was not found. please try again later"

Typo3 generates to erros in the log:
"Core: Error handler (FE): PHP Warning: Cannot modify header information - headers already sent in /path/to/typo3/sysext/frontend/Classes/Controller/TypoScriptFrontendController.php line 3854 "
and
"Core: Error handler (FE): PHP Warning: Cannot modify header information - headers already sent in /path/to/typo3/sysext/frontend/Classes/Utility/CompressionUtility.php line 55 "

Any idea what's wrong?

**Comment by derhansen** [comment]
This might be an issue in the TYPO3 core, when FE compression is enabled (at least for BE compression the bug has been fixed - see https://forge.typo3.org/issues/64330). 

Could you try to set ```$TYPO3_CONF_VARS['FE']['compressionLevel'] = 0```and give me feedback if this works for you.

**Comment by lorddoumer** [comment]
thanks for the fast feedback, this solves the problem indeed!
so I have to leave compression disabled for typo3 6.2?

**Comment by derhansen** [comment]
yes, at the moment you should leave it disabled. I'll create a ticket on forge for this issue.

**Comment by lorddoumer** [comment]
Ok, since the problem is linked to the cms core, I close this issue.
thanks for your help!

**Comment by derhansen** [comment]
I'll reopen this as a reminder for me.

**Comment by derhansen** [comment]
Try to find alternative for dumpFileContent
